docs/advanced/mixin/4.shadow.md

```diff
@@ -0,0 +1,46 @@
+@Shadow
+======
+
+Shadow is a technology that redirects the access of the fields and methods from the mixin classes to the vanilla classes. It allows you to use those elements like normal ones so that your method code can be simplified. It can also help you to access the private fields or methods in the target class. 
+
+Fields
+------
+Shadowed fields are annotated with  ```@Shadow``` and have the same access modifiers, names, and types as the vanilla fields in the target class. 
+
+```java
+@Shadow
+<access modifier> <static or not> <type name> <field name>;
+```
+
+The Mixin system will automatically redirect your access to the corresponding fields in the target class.
```

Comment:
Not really true since it depends on other compatibilities. Edge cases should be documented.